libs/designsystem/slide/src/slides.component.ts

```diff
@@ -1,54 +1,111 @@
-import { EventEmitter } from '@angular/core';
 import {
-  ChangeDetectionStrategy,
+  AfterViewInit,
   Component,
   ContentChild,
+  EventEmitter,
   Input,
+  OnInit,
   Output,
   TemplateRef,
   ViewChild,
 } from '@angular/core';
-import { IonSlides } from '@ionic/angular';
-
+import { SwiperOptions } from 'swiper';
+import { register } from 'swiper/element/bundle';
+import { PlatformService, UniqueIdGenerator } from '@kirbydesign/designsystem/helpers';
 import { SlideDirective } from './slide.directive';
 
+// Swiper is not an Angular library,
+// so we need to use their web components and register them manually.
+// https://swiperjs.com/element
+register();
+
+export type SelectedSlide = {
+  slide: any;
+  index: number;
+};
+
+export type KirbySwiperOptions = SwiperOptions;
 @Component({
   selector: 'kirby-slides',
-  template: `
-    <ion-slides
-      [options]="slidesOptions"
-      #ionslides
-      (ionSlideDidChange)="onSlideChanged()"
-      ionSlidesDidLoad="slideTo(0)"
-    >
-      <ion-slide *ngFor="let slide of slides; let i = index">
-        <ng-container
-          *ngTemplateOutlet="slideTemplate; context: { $implicit: slide, index: i }"
-        ></ng-container>
-      </ion-slide>
-    </ion-slides>
-  `,
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './slides.component.html',
+  styleUrls: ['./slides.component.scss'],
 })
-export class SlidesComponent {
-  @ViewChild('ionslides', { static: false }) ionSlides: IonSlides;
-  @Input() slidesOptions: any;
-  @Input() slides: any[];
-  @Output() selectedSlide = new EventEmitter<any>();
+export class SlidesComponent implements OnInit, AfterViewInit {
+  constructor(private platform: PlatformService) {}
 
+  @ViewChild('swiperContainer') swiperContainer;
   @ContentChild(SlideDirective, { static: true, read: TemplateRef })
   public slideTemplate: TemplateRef<any>;
 
-  onSlideChanged() {
-    this.ionSlides.getActiveIndex().then((selectedIndex) => {
-      this.selectedSlide.emit({
-        slide: this.slides[selectedIndex],
-        index: selectedIndex,
-      });
-    });
+  @Input() slidesOptions: KirbySwiperOptions;
+  @Input() ignorePagePadding: boolean;
+  @Input() title: string;
+  @Input() slides: any[];
+
+  // simpleSlider is a temporary solution to make the slides component backwards compatible with the old design.
+  @Input() simpleSlider = false;
+
+  @Output() selectedSlide = new EventEmitter<SelectedSlide>();
```

Comment:
Consider deprecating this naming and adding a new output prop that matches both Kirby and swiperjs terminology: 
```suggestion
  /**
   * @deprecated Will be removed in next major version. Use `slideChange` instead.
   */
  @Output() selectedSlide = new EventEmitter<SelectedSlide>();
  @Output() slideChange = new EventEmitter<SelectedSlide>();
```